doc/tutorials/application-integration.md

```diff
@@ -205,39 +200,39 @@ For more information about that, continue reading
 
 ## Specification ##
 
-Now, we have a fully working configuration system without any hard-coded
-information (such as configuration files). So we already gained something.
+Now, we have a fully working configuration system without any hardcoded
+information (such as configuration files). We already gained something.
 But, we did not discuss how we can actually achieve application integration,
 the goal of Elektra.
 
 Elektra 0.8.11 introduces the so called specification for the
-application's configuration. It is located below its own [namespace](/doc/help/elektra-namespaces.md)
+application's configuration. It's located below its own [namespace](/doc/help/elektra-namespaces.md)
 `spec` (next to user and system).
 
-Keys in `spec` allow us to specify which keys are read by the application,
+Keys in `spec` allow us to specify which keys the application reads,
 which fallback they might have and which is the default value using
 metadata. The implementation of these features happened in `ksLookup`.
-When cascading keys (those starting with `/`) are used following features
+When using cascading keys (those starting with `/`), the following features
 are now available (in the metadata of respective `spec`-keys):
 
-- `override/#`: use these keys *in favour* of the key itself (note that
+- `override/#`: use these keys *in favor* of the key itself (note that
     `#` is the syntax for arrays, e.g. `#0` for the first element,
     `#_10` for the 11th and so on)
 - `namespace/#`: instead of using all namespaces in the predefined order,
-    one can specify which namespaces should be searched in which order
+    one can specify namespaces to search in a given order
 - `fallback/#`: when no key was found in any of the (specified) namespaces
     the `fallback`-keys will be searched
-- `default`: this value will be used if nothing else was found
+- `default`: the value to use if nothing else was found
 
-They can be used like this:
+You can use those features like following:
 
 ```sh
 kdb set /overrides/test "example override"
 sudo kdb setmeta spec/test override/#0 /overrides/test
 ```
 
-This technique provides complete transparency how a program will fetch a configuration
-value. In practice that means that:
+This technique provides complete transparency how a program will fetch a
+configuration value. In practice that means that:
```

Comment:
Please separate formatting changes with other changes in future. It is difficult to read this PR.

(At least different commits!)